### PR TITLE
Updated MroCtxIsisLabelNaifSpiceDriver and Tests

### PR DESCRIPTION
**Pull request by paarongiroux**

### CODE REVIEW

File: ale/base/label_isis.py [hunk 1]
```diff
@@ -134,3 +134,7 @@ def  exposure_duration(self):
             # if no units are available, assume the exposure duration is given in milliseconds
             exposure_duration = self.label['IsisCube']['Instrument']['ExposureDuration'].value * 0.001
         return exposure_duration
+
+    @property
+    def line_exposure_duration(self):
```
Comment (jessemapel) [inline ale/base/label_isis.py]: Needs doc string

**Comment by jessemapel** [comment]
Fixes #152

File: ale/base/label_isis.py [hunk 2]
```diff
@@ -134,3 +134,15 @@ def  exposure_duration(self):
...
+
+    @property
+    """
+    The line exposure duration of the image, in seconds
+
+    Returns
+    -------
+    : float
+      Line exposure duration in seconds
+    """
```
Comment (jessemapel) [inline ale/base/label_isis.py]: This needs to go under the `def line_exposure_duration` line

Comment (paarongiroux) [reply ale/base/label_isis.py]: yep, just realized that

**Pull request merged**